argo/overlays/docker-for-desktop/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+bases:
+- ../../base
```

Comment:
Do we need to define an overlay if the overlay isn't doing any customization?

Reply:
I thought I'd get an initial commit in and add the overlays specific to each platform later

Reply:
I wouldn't expect most packages to require platform specific customization.
So I would only expect us to define overlays for those packages and platforms requiring customization.

But this PR seems to be implying that we will define overlays for every platform for every package because you are creating stubs to be filled out later.

Reply:
Oh yeah, that makes sense. I'll remove stubs for components that don't require customization.